lib/server.go

```diff
@@ -217,8 +218,22 @@ func (srv *Server) GetBlockchain() *Blockchain {
 	return srv.blockchain
 }
 
+type BackendMempool interface {
```

Comment:
Now that we've seen the new interface expressed in the code, it's worth just merging this interface with the existing `Mempool` interface. You can have the existing DeSoMempool struct just stub out the interface methods that we don't want/need it to support. The stubbed out methods can return a "Not Implemented" error

Reply:
Deal!!